dog_lover/components/dogGeneratorBox.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FunctionComponent, useEffect } from "react";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import {
+  useVariableRandomImageContext,
+  useFunctionRandomImageContext,
+} from "../src/breedContext";
+// import "aos/dist/aos.css";
```

Comment:
It this needed here?

Reply:
Ah yes, these comments that are left along with the respective package in the package.json, is my *failed* attempt to add it to my Box components to make them more interactive if I may say so, or just animate them. I loved this package when I was designing my own website and wanted to use it here as well, but it was acting up! I think the latest version of it is in desperate need of an update/bug fix! And of course, leaving those comments was actually on purpose because I really wanted to talk about this package hahaha it made my life so much easier when I wanted to animate my website!